docs/_toc.yml

```diff
@@ -16,12 +16,10 @@ subtrees:
       - file: tutorials/index
         subtrees:
         - entries:
+          - file: tutorials/annotation/index
+          - file: tutorials/processing/index
           - file: tutorials/segmentation/index
           - file: tutorials/tracking/index
```

Comment:
```suggestion
          - file: tutorials/annotation/index
            subtrees:
            - entries: 
              - file: tutorials/annotation/annotate_points
          - file: tutorials/processing/index
            subtrees:
            - entries:
              - file: tutorials/processing/dask
          - file: tutorials/segmentation/index
            subtrees:
            - entries:
              - file: tutorials/segmentation/annotate_segmentation
          - file: tutorials/tracking/index
            subtrees:
            - entries:
              - file: tutorials/tracking/cell_tracking
```

Reply:
The last error should be fixed in L98 of the points how-to: the link to the point annotation tutorial should be
```
[point annotation tutorial](../../tutorials/annotation/annotate_points)
```

Reply:
got it, thanks!